src/auth/jwtaccess.ts

```diff
@@ -71,12 +74,9 @@ export class JWTAccess {
     // The payload used for signed JWT headers has:
     // iss == sub == <client email>
     // aud == <the authorization uri>
-    const payload = {iss: this.email, sub: this.email, aud: authURI, exp, iat};
-    const assertion = {
-      header: {alg: 'RS256'} as jws.Header,
-      payload,
-      secret: this.key
-    };
+    const payload = Object.assign(
+        {iss: this.email, sub: this.email, aud: authURI, exp, iat},
+        additionalClaims);
```

Comment:
What's the behavior you want to have when the user has a `iss` property in `additionalClaims`? Are they allowed to override `this.email`?

I think you should decide on a behavior, document it, and add some tests.

Reply:
Decided to support overriding, and add a test specifically for it.  Good call 👍

Reply:
Reverse that, see comment below 😂